app/src/pages/EditPage/EditPage.scss

```
@@ -0,0 +1,7 @@
+@import '~/assets/scss/mixins/_typo.scss';
+
+.editPage {
```

классы в css через кебаб
```suggestion
.edit-page {
```